demo/views/demo-all.dust

```diff
@@ -34,14 +35,11 @@
   </div>
 
   <!-- Scripts -->
-  <!-- <script src="../../dist/bluemix-components.min.js"></script> -->
-  <script src="/demo.js"></script>
+  <script src="/carbon-components.min.js"></script>
+  <!-- <script src="/demo.js"></script> -->
```

Comment:
Oh, and the problem I ran into was when I had to deploy the app to Bluemix for reviewing purposes. It would not import the javascript properly.